core-web/libs/dot-primeng-theme-styles/src/scss/_colors.scss

```diff
@@ -0,0 +1,56 @@
+// This file contain all the colors from the design system of dotCMS
```

Comment:
what's the intention of these file? is that something only for experiments?

I think on the variable.scss file we have some of those vars
![image](https://user-images.githubusercontent.com/37185433/214646146-cfbf5d1e-3a77-4818-8b0b-6fa075c40b6d.png)